packages/block-library/src/embed/index.js

```diff
@@ -163,39 +166,26 @@ export function getEmbedEdit( title, icon ) {
 
 			if ( iframe && iframe.height && iframe.width ) {
 				const aspectRatio = ( iframe.width / iframe.height ).toFixed( 2 );
-				let aspectRatioClassName;
-
-				switch ( aspectRatio ) {
+				const aspectRatios = [
```

Comment:
This seems like it fits criteria of being a constant:

https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#constants

Reply:
Screaming snake case is my favourite case name. Changed :)